### ISSUE TITLE
[Features] Terraform approaches for builtin:process.built-in-process-monitoring-rule

### ISSUE THREAD

**Issue by JayChanggithub** [dynatrace-oss/terraform-provider-dynatrace#274]
Hi Team, 
We had a implementation of rest API like below, Just confirmed have any possible to adopt terraform approaches to do the same feature. if so we need to adopt which modules to do? Thanks

```
curl '[https://apm.cf.eu20.hana.ondemand.com/e/3ddcb8bf-6657-46b1-ab6a-4000eb090d82/api/v2/settings/objects'](https://apm.cf.eu20.hana.ondemand.com/e/3ddcb8bf-6657-46b1-ab6a-4000eb090d82/api/v2/settings/objects%27) \
-X POST \
-H 'Accept: application/json; charset=utf-8' \
-H 'Content-Type: application/json; charset=utf-8' \
-H 'Authorization: Api-Token ACCESS_TOKEN' \
-d $'[{"schemaId":"builtin:process.built-in-process-monitoring-rule","scope":"tenant","value":{"-2":true,"-3":true,"-4":true,"-49":true,"-50":true,"-51":true,"-52":true,"-53":true,"-54":true,"-47":false,"-5":true,"-6":true,"-7":true,"-8":true,"-9":true,"-10":true,"-11":true,"-12":true,"-13":true,"-14":true,"-16":true,"-17":true,"-18":true,"-19":true,"-20":true,"-21":true,"-22":true,"-23":true,"-24":true,"-25":true,"-26":true,"-27":true,"-28":true,"-29":true,"-45":true,"-68":true,"-32":true,"-33":true,"-34":true,"-35":true,"-36":true,"-55":true,"-56":true,"-43":true,"-37":true,"-38":true,"-39":true,"-44":true,"-58":true,"-67":true,"-69":true,"-70":true,"-40":true,"-41":true,"-46":true,"-48":true,"-57":true,"-59":true,"-60":true,"-61":true,"-62":true,"-63":true,"-64":true,"-65":true,"-66":true}}]'
```

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Hello @JayChanggithub,
Are you specifically talking about a resource that covers the schema `builtin:process.built-in-process-monitoring-rule`?
Or is the idea to offer a resource that allows for addressing any schema of the Settings API generically?

**Comment by JayChanggithub** [comment]
Hi @Dynatrace-Reinhard-Pilz  
We wanna to adopt terraform approaches to implememt schema builtin:process.built-in-process-monitoring-rule. Yes. Have any resource to cover? Since we adopted API method to do previosly. But we wanna to update from API to terraform to manage those resources. If so could you point what resources of terraform to do it?

**Comment by kishikawa12** [comment]
Hi @JayChanggithub,

The next release will introduce the resource `dynatrace_builtin_process_monitoring` which will allow you to configure the built-in process monitoring rules. 

For readability and usability, I opted to introduce booleans with an abbreviated name instead of the rule number. They are all tagged as optional with default true, so all you are required to do is define the values you would like to drive to false. All of the fields will be available in the Terraform Registry [documentation](https://registry.terraform.io/providers/dynatrace-oss/dynatrace/latest/docs) once released.

Example:

```
resource "dynatrace_builtin_process_monitoring" "#name#" {
  host_group_id = "environment"
  go_static = false
}
```

We're looking to either push a release Wed/Thurs this week, I hope that helps!

**Comment by JayChanggithub** [comment]
@kishikawa12  
Thanks.

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Fixed with [v1.33.0](https://github.com/dynatrace-oss/terraform-provider-dynatrace/releases/tag/v1.33.0)
